connectors/sources/tests/test_mysql.py

```diff
@@ -303,15 +293,15 @@ async def test_fetch_rows_from_tables():
 
     # Execute
     with mock.patch.object(
-        aiomysql, "create_pool", return_value=(await mock_mysql_response())
+        source, "with_connection_pool", return_value=ConnectionPool()
     ):
         response = source._connect(query)
 
         mock.patch("source._connect", return_value=response)
 
-    # Assert
-    async for row in source.fetch_rows_from_tables("table"):
-        assert "_id" in row
+        # Assert
```

Comment:
```suggestion
```

I'm not the biggest fan of the `# Setup`, `# Execute` and `# Assert` comments tbh. Feel free to keep it for consistency, but I think we can remove them when we see them.

Reply:
Agreed!